database/zmon/20_api/05_stored_procedures/60_onetime_access_tokens.sql

```diff
@@ -26,6 +26,8 @@ $$
    WHERE oat_token = token
      AND (oat_bound_ip is NULL OR oat_bound_ip = bind_ip)
      AND (oat_bound_session_id is NULL or oat_bound_session_id = session_id)
+     -- token only valid for 24h - assigned session extends until bound_expires
+     AND (oat_valid_until > NOW() OR oat_bound_session_id = session_id)
```

Comment:
Hm...  I think mixing "valid until" and "bound expires" in one table is already confusing.  Would it not make more sense to have `bounds` as a separate table?